src/azul/indexer/document.py

```diff
@@ -651,6 +658,159 @@ def filter(self, relation: str, values: list[AnyJSON]) -> list[JSON]:
         return result
 
 
+class ValueAndUnit(FieldType[JSON, str]):
```

Comment:
It's unclear to me why moving ValueAndUnit is part of this PR. Please remove or explain.

Reply:
It's to avoid a circular import. This doesn't show up in the unit tests, but I try deploying without the first commit I get:

```
make -C terraform plan auto_apply
make[1]: Entering directory '/home/nadove/PycharmProjects/azul-sc/terraform'
...
Creating sqs.tf.json
python step_function.tf.json.template.py step_function.tf.json
Traceback (most recent call last):
  File "/home/nadove/PycharmProjects/azul-sc/terraform/step_function.tf.json.template.py", line 12, in <module>
    from azul.service.manifest_controller import (
  File "/home/nadove/PycharmProjects/azul-sc/src/azul/service/manifest_controller.py", line 38, in <module>
    from azul.service.manifest_service import (
  File "/home/nadove/PycharmProjects/azul-sc/src/azul/service/manifest_service.py", line 109, in <module>
    from azul.service import (
  File "/home/nadove/PycharmProjects/azul-sc/src/azul/service/avro_pfb.py", line 56, in <module>
    from azul.plugins.metadata.hca.indexer.transform import (
  File "/home/nadove/PycharmProjects/azul-sc/src/azul/plugins/metadata/hca/__init__.py", line 49, in <module>
    from azul.service.manifest_service import (
ImportError: cannot import name 'ManifestFormat' from partially initialized module 'azul.service.manifest_service' (most likely due to a circular import) (/home/nadove/PycharmProjects/azul-sc/src/azul/service/manifest_service.py)
```